tests/tests/fv3_gsd_drag_suite

```diff
@@ -76,7 +76,7 @@ export_fv3
 export FHMAX=24
 export FDIAG=3
 
-export DT_ATMOS="600"
+export DT_ATMOS="450"
```

Comment:
I am curious what is the reason to reduce the time step in this PR? I seem not seeing any code changes that requires this change.

Reply:
This is unrelated to the diagnostics. The gsd physics suite parallels needed a smaller timestep to avoid crashes, and we're updating the regression tests to match.

Reply:
Yes.